EnglishTranslation-master/ITEM.tsv

```diff
@@ -10195,44 +10195,44 @@ ITEM_20160224_010194	Rune Stone of Destruction
+ITEM_20160310_010211	Some people believe that Lolopanther equipment is made without the permission of the goddess, and simply taking advantage of her absence.
 ITEM_20160310_010212	Others believe Goddess Zemyna bestowed the right to freely use any material that grows on her land. If that wasn't the case, it would be impossible to mine the materials. However, this does not mean that we have the absolute right to do what we want.
 ITEM_20160310_010213	The history of the Lolopanthers is surprisingly long. Unlike other weapons, the Lolopanther Dagger dates back to before the kingdom was founded.
 ITEM_20160310_010214	One legend told that Goddess Zemyna's temple itself was a pillar of the earth. The legend is now forgotten, but the staff celebrating the legend remains to this day.
 ITEM_20160310_010215	Warsis Creak has succeeded in developing a new type of weapon from the Lolopanther Musket. However, it is unknown when and how this happened.
-ITEM_20160310_010216	A weapon firm as the ground, light as a feather and strong as a rock.
+ITEM_20160310_010216	A weapon as firm as the ground, light as a feather, and strong as a rock.
 ITEM_20160310_010217	The masters who deal with gunpowder regularly use it with newly designed or discovered weapons. The same can be said for this cannon. Quicktongue, however, hardly shares any new discoveries and is likely considering his military career.
 ITEM_20160310_010218	Those who dare to belittle the weapons grown by the goddess are doomed to have their blood fed to her earth.
-ITEM_20160310_010219	Those who worship the great goddess will receive wisdom of humility. This equipment proves that this is true not only for clerics, but for wizards too.
-ITEM_20160310_010220	Whoever carries this will become the master of the earth and receive its grace with every step. However, maintaining the cost of this equipment is of course a problem for humans and unrelated to the goddess.
-ITEM_20160310_010221	Regardless of what rights the goddess of the earth may give, those who carry and use this equipment do so with their own abilities. Do you not wonder whether it possesses an even greater power?
-ITEM_20160310_010222	With every new piece of equipment, the power bestowed by the goddess of the earth is newly imparted on it.
-ITEM_20160310_010223	Users of this equipment may not realize, but designers and artisans know that obtaining the blessing of Goddess Zemyna for using leather is not as easy as it is for using cloth or metal. But thanks to their effort it shines today on the battlefield.
+ITEM_20160310_010219	Those who worship the great goddess will understand humility. This equipment proves that this is true not only for clerics, but for wizards too.
+ITEM_20160310_010220	Whoever carries this will become the master of the earth and receive its grace with every step. However, maintaining this equipment is a problem for humans in regard to cost, and of no concern to the goddess.
```

Comment:
`maintaining this equipment is a problem for humans in regard to cost, and of no concern to the goddess.`
to
`maintaining this equipment is a problem for humans in regards to cost, which is of no concern to the goddess.`